pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -583,7 +587,7 @@ func Run(ps *pubsub.PubSub) {
 		initializeSelfPublishHandles(ps, &ctx)
 
 		fscryptStatus, fscryptErr := fetchFscryptStatus()
-		publishVaultStatus(&ctx, defaultImgVaultName, defaultImgVault,
+		publishVaultStatus(&ctx, defaultImgVaultName, defaultVault,
```

Comment:
Please rename defaultImgVaultName to defaultVaultName to match the other constant names.